.changeset/forty-beers-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-catalog-graph': patch
+---
+
+Added possibility to show arrow heads for graph edges in `DependencyGraph` for better understandability.
```

Comment:
Can you include just a small code sample of how this gets used here, please? These changesets get rolled up in the the changelog for each release so having a basic example here means if this is a feature you want you can easily implement it.

Reply:
Right, on it. Let me know if you'd also like to adjust the wording.